dracut.sh

```diff
@@ -961,6 +935,62 @@ for f in $(dropindirs_sort ".conf" "$confdir" "$dracutbasedir/dracut.conf.d"); d
     [[ -e $f ]] && . "$f"
 done
 
+# regenerate_all shouldn't be set in conf files
+regenerate_all=$regenerate_all_l
+if [[ $parallel_l == "yes" ]]; then
+        parallel=yes
+fi
+
+if [[ $regenerate_all == "yes" ]]; then
+    ret=0
+    if [[ $kernel ]]; then
+        printf -- "--regenerate-all cannot be called with a kernel version\n" >&2
+        exit 1
+    fi
+
+    if [[ $outfile ]]; then
+        printf -- "--regenerate-all cannot be called with a image file\n" >&2
+        exit 1
+    fi
+
+    ((len = ${#dracut_args[@]}))
+    for ((i = 0; i < len; i++)); do
+        case ${dracut_args[$i]} in
+            --regenerate-all | --regenerate-parallel)
```

Comment:
You forgot to change `--regenerate-parallel`  to `-p`, `--parallel` here

Reply:
right, thanks. But as `--parallel` is supposed to be independent of `--regenerate-all`, maybe I should pass the flag down to the child processes instead? (currently it won't have any effect as `--regenerate-all` is removed).

Reply:
re-pushed; still removing `--parallel`. Tell me what you prefer.

Reply:
Let's keep this as is